admin_sdk/reseller/quickstart.py

```diff
@@ -14,41 +14,40 @@
 
 # [START admin_sdk_reseller_quickstart]
 """
-Shows basic usage of the Admin SDK Reports API. Outputs a list of last 10 login
-events.
+Shows basic usage of the Admin SDK Reseller API. Prints the customer ID, SKU ID,
```

Comment:
I noticed that the Reseller quickstart had the Reports API content in it, probably a copy/paste mistake. Restored the original Reseller content.